### ISSUE TITLE
No correct bounding boxes

### ISSUE THREAD

**Issue by Cbanyungong** [experiencor/keras-yolo2#300]
the loss is always declining, but the accuracy dose not rise. When detecting Blood Cell,confidences of all boxed are 0.5 there is no correct boxes.Could you tell me what my mistake is?And the loss of WH is very high.

Loss XY 	[0.0567559749]
Loss WH 	[5.61224461]
Loss Conf 	[0.133079857]
Loss Class 	[0]
Total Loss 	[5.80208063]
Current Recall 	[0.937499881]
Average Recall 	[0.892882764]

![cell](https://user-images.githubusercontent.com/29659406/40869977-d537b2b6-6657-11e8-8d0a-eda8174a5d9b.png)

**Comment by knightlyw** [comment]
me too, I think the loss function is wrong, it's different from the formula

**Comment by knightlyw** [comment]
@Cbanyungong  Don't do average

**Comment by Cbanyungong** [comment]
Thanks for your perfect work,I got the correct results

**Comment by 2016JTM2091** [comment]
@knightlyw  Can you please help how to make the loss function correct? I'm facing the same issue.

**Comment by E-LEAF** [comment]
@knightlyw I'm facing the same problem. Can you tell how to avoid averaging in the loss function
